certs/api/endpoint.go

```diff
@@ -18,9 +18,15 @@ func issueCert(svc certs.Service) endpoint.Endpoint {
 		}
 		res, err := svc.IssueCert(ctx, req.token, req.ThingID, req.Valid, req.KeyBits, req.KeyType)
 		if err != nil {
-			return certsResponse{Error: err.Error()}, nil
+			return thingCertsRes{Error: err.Error()}, nil
 		}
-		return res, nil
+		return thingCertsRes{
+			Serial: res.Serial,
+			ID:     res.ThingID,
+			Key:    res.ClientKey,
+			Cert:   res.ClientCert,
+			CACert: res.IssuingCA,
```

Comment:
Is there another CA then issuing CA?

Reply:
no, but since the response from `vault` names this `issuing_ca` I used that naming